### PR TITLE
INBT-1551 | Change document checking route to the new Politsei.ee route

### PR DESCRIPTION
**Pull request by Screatch**
This pull request refactors existing functionality and removes need to do a POST a form.
Now all information is accessible one GET request away.

I also fixed tests and did a version bump.
This is supposed to be used only in Cofi-HP product, so next part I will switch it in Cofi and see if everything still works.

### CODE REVIEW

File: spec/validator_spec.rb [hunk 1]
```diff
@@ -64,51 +55,49 @@
...
   private
 
-  def mock_response_with(type, response)
-    allow(RestClient).to receive(type) { send("#{response}_response") }
-    allow_any_instance_of(Validocno::Validator).
-      to receive(:check_response) { true }
+  def mock_response_with(response)
+    response_result = send("#{response}_response")
+
+    stub_request(:get, "https://www.politsei.ee/qr/?qr=#{response_result[:document_number]}").
```
[the remark this answers is not in the document]
Comment (Screatch) [reply spec/validator_spec.rb]: Well it's URL, I don't have many options to split the URL here so I would say this is irrelevant.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 require 'validocno'
 require 'webmock/rspec'
 
-WebMock.disable_net_connect!(:allow_localhost => true)
+WebMock.disable_net_connect!(allow_localhost: true)
```
[the remark this answers is not in the document]
Comment (Screatch) [reply spec/spec_helper.rb]: If you add new line, it will start complaining, pls ignore.

File: spec/validator_spec.rb [hunk 3]
```diff
@@ -64,51 +55,57 @@
...
+      with(:headers => {
+          "Accept" => "*/*",
+          "Accept-Encoding" => "gzip, deflate",
+          "Host" => "www.politsei.ee",
+          "User-Agent" => "rest-client/2.0.2 (darwin17.2.0 x86_64) ruby/2.1.10p492"
+      }).to_return(
+        :status => 200,
+        :body => response_result[:body],
+        :headers => {}
```
[the remark this answers is not in the document]
Comment (Screatch) [reply spec/validator_spec.rb]: Thanks, but rather not.

File: .gitignore [hunk 4]
```diff
@@ -12,3 +12,4 @@
 *.o
 *.a
 mkmf.log
+/.idea
```
Comment (defeed) [inline .gitignore]: I think these kind of files belong in your global `.gitignore`, not here

File: lib/validocno/validator.rb [hunk 5]
```diff
@@ -1,57 +1,34 @@
 module Validocno
   class Validator
-    # Set strings to UTF-8 mode
-    $KCODE = 'UTF-8'
 
-    CONTROL_URI = 'https://www.politsei.ee/et/teenused/e-paringud/dokumendi-kehtivuse-kontroll/'
+    CONTROL_URI = 'https://www.politsei.ee/qr/'.freeze
```
Comment (defeed) [inline lib/validocno/validator.rb]: Double quotes please

File: spec/spec_helper.rb (hunk 2 above)
Comment (defeed) [inline spec/spec_helper.rb]: Why did you change it? This whole thing uses old hash syntax everywhere :)

File: .gitignore (hunk 4 above)
Comment (Screatch) [reply .gitignore]: Makes sense, I didn't know of global .gitignore. Removed it from there.

File: spec/spec_helper.rb (hunk 2 above)
Comment (Screatch) [reply spec/spec_helper.rb]: Missed that, I was following hound advices until I realized that the project uses old version of ruby and I was running tests on newer version of ruby. Fixed now.
